doc/software_support.md

```diff
@@ -32,6 +32,7 @@ For all browsers and to track browsers progress see [Can I Use](https://caniuse.
 - [GDAL](https://gdal.org/drivers/raster/jpegxl.html): supported since 3.4.0 as a TIFF codec, and 3.6.0 as standalone format
 - [GraphicsMagick](http://www.graphicsmagick.org/NEWS.html#march-26-2022): supported since 1.3.38
 - [SAIL](https://sail.software): supported since 0.9.0
+- [Jpeg XL coder for Android 21+](https://github.com/awxkee/jxl-coder)
```

Comment:
```suggestion
- [JPEG XL coder for Android 21+](https://github.com/awxkee/jxl-coder)
```

Reply:
great! My suggested edit was about the capitalization of "JPEG"

Reply:
Thank you for the feedback. I fixed the capitalization based on your suggestion.